src/jwt-middleware.js

```diff
@@ -89,7 +89,7 @@ function middleware() {
     authenticate(req, res).
       then((jwt) => {
         if (!jwt) {
-          res.sendStatus(401);
+          res.status(401).end();
```

Comment:
Is there a difference here?

Reply:
`sendStatus()` includes the actual error text. That should be fine, but it ended up causing test failures, so I switched it.